### PR TITLE
Redirect localkube logs in sudo

### PR DESCRIPTION
**Pull request by jimmidyson**
Required for #301

### CODE REVIEW

File: pkg/minikube/cluster/commands.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var stopCommand = "sudo killall localkube | true"
 
 var startCommandFmtStr = `
 # Run with nohup so it stays up. Redirect logs to useful places.
-PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube %s --generate-certs=false --logtostderr=true > %s 2> %s < /dev/null &
+sudo sh -c 'PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube %s --generate-certs=false --logtostderr=true > %s 2> %s < /dev/null &'
```
Comment (dlorenc) [inline pkg/minikube/cluster/commands.go]: Do we still need the inner "sudo"?

Comment (luxas) [inline pkg/minikube/cluster/commands.go]: Don't think so

Comment (jimmidyson) [inline pkg/minikube/cluster/commands.go]: No missed that - thanks for spotting it!

Comment (jimmidyson) [inline pkg/minikube/cluster/commands.go]: Removed.

**Comment by dlorenc** [comment]
Nice!

**Pull request merged**